.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```

Comment:
Where would this issue template be used and is this relevant to this PR?

Reply:
It persists the current functionality (image below ) while also adding a link to Discord server as an option for help

![image](https://user-images.githubusercontent.com/634958/119181385-86ab2080-ba3f-11eb-8e5f-3dc933c3cce5.png)